### PR TITLE
Use logging to print messages.

### PR DESCRIPTION
**Pull request by zheng-da**
*Description of changes:*
We should use `logging` to print messages so that we can set the logging level to hide some messages.

Fix the issue: https://github.com/awslabs/graphstorm/issues/204

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: python/graphstorm/gconstruct/construct_graph.py [hunk 1]
```diff
@@ -430,7 +449,12 @@ def process_graph(args):
         edges = edges1
         sys_tracker.check('Add reverse edges')
     g = dgl.heterograph(edges, num_nodes_dict=num_nodes)
-    print(g)
+    logging.info("The graph has %d node types and %d edge types.",
+                 len(g.ntypes), len(g.etypes))
+    for ntype in g.ntypes:
+        logging.info("Node type %s has %d nodes.", ntype, g.number_of_nodes(ntype))
+    for etype in g.canonical_etypes:
+        logging.info("Edge type %s has %d edges.", str(etype), g.number_of_edges(etype))
```
Comment (classicsong) [inline python/graphstorm/gconstruct/construct_graph.py:457]: What about train/val/test info?

**Review by classicsong — APPROVED** [review]
One comment.
Overall LGTM

File: python/graphstorm/gconstruct/construct_graph.py (hunk 1 above)
Comment (zheng-da) [reply python/graphstorm/gconstruct/construct_graph.py:457]: print out train/val/test split?

Comment (zheng-da) [reply python/graphstorm/gconstruct/construct_graph.py:457]: then we should print out features as well.

**Pull request merged**